crates/hir/src/lib.rs

```diff
@@ -141,7 +139,7 @@ pub use {
         },
         hygiene::{SyntaxContextExt, marks_rev},
         inert_attr_macro::AttributeTemplate,
-        mod_path::tool_path,
+        mod_path::{ModPath, PathKind, tool_path},
```

Comment:
Why are these added? I don't see them being used. And why are the negative reexports removed?

Reply:
They were re-exported before, just from `path` not `mod_path`